docs/persistence.md

```
@@ -0,0 +1,19 @@
+# Persistence
+
+![](./figures/lib/lib-persistence.png)
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)